### PR TITLE
DeviceRow: If name is null, try to fall back to type

### PR DESCRIPTION
**Pull request by danrabbit**
If we can get the device type from the icon name, I think that's a better fallback than going straight to the device address

### CODE REVIEW

**Comment by tintou** [comment]
Maybe still provide the address within parenthesis as there is then no way to differentiate two devices…

File: src/DeviceRow.vala [hunk 1]
```diff
@@ -86,7 +86,40 @@ public class Bluetooth.DeviceRow : Gtk.ListBoxRow {
         overlay.add (image);
         overlay.add_overlay (state);
 
-        var label = new Gtk.Label (device.name ?? device.address);
+        string device_name = device.name;
```
Comment (tintou) [inline src/DeviceRow.vala:89]:
```suggestion
        unowned string device_name = device.name;
```

**Comment by danrabbit** [comment]
@tintou addresses are shown in tooltips :)

**Comment by tintou** [comment]
> @tintou addresses are shown in tooltips :)

Fair enough then :)

**Review by tintou — APPROVED** [review]

**Pull request merged**